index.html

```diff
@@ -189,6 +189,34 @@ <h4>Skills</h4>
             </ul>
         </section>
     </section>
+    <section class="contact">
+        <div class="contact-wrap">
+            <img src="images/contact_shapes.png" alt="" srcset="" class="contact-shapes-bg">
+            <h4 class="contact-title">Contact me</h4>
+            <p class="contact-desc">
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding.
+                I’d love to help with it!
+            </p>
+            <form action="https://formspree.io/f/xwkjrjye" method="post" class="contact-form">
+                <ul>
+                    <li>
+                        <input type="text" id="name" name="user_name" value="Lorem Ipsum" maxlength="30" />
+                    </li>
```

Comment:
- [ ] Nice work implementing the changes :wink: However we should use `placeholder="Lorem Ipsum"` instead of `value="Lorem Ipsum"` as requested in the project requirement (see image).

- [ ] I was able to submit the form while the name field is empty. Kindly add `required` attribute to the name `input` to prevent that as requested in the project requirement (see image).
![Screenshot from 2023-04-05 11-22-05](https://user-images.githubusercontent.com/91142432/230066261-7db1fd5d-5153-4d6d-9ad0-aa170493749e.png)